packages/victory-axis/src/helper-methods.js

```diff
@@ -296,12 +372,28 @@ const getCalculatedValues = (props) => {
   const stringTicks = Axis.stringTicks(props) ? props.tickValues : undefined;
   const axis = Axis.getAxis(props);
   const orientation = getOrientation(props);
-  const scale = getScale(props);
-  const domain = Axis.getDomain(props);
-  const ticks = Axis.getTicks(props, scale, props.crossAxis);
-  const tickFormat = Axis.getTickFormat(props, scale);
+  const currentAxisScale = getScale(props);
+  const ticks = Axis.getTicks(props, currentAxisScale, props.crossAxis);
+  const tickFormat = Axis.getTickFormat(props, currentAxisScale);
   const anchors = getAnchors(orientation, isVertical);
 
+  const range = {
+    x: Helpers.getRange(props, "x"),
+    y: Helpers.getRange(props, "y")
+  };
+  const domain = {
+    x: Domain.getDomain(props, "x"),
```

Comment:
I'm not sure `Domain.getDomain` is going to do the same thing as `Axis.getDomain`, since the generic `getDomain` doesn't care about `tickValues`.  I think `Axis.getDomain` takes an optional `axis` prop (i.e. x or y), so I think you can drop in replace that here.

Reply:
Thinking about this more, I think the correct behavior is actually something like:

```
domain = {
  x:  props.domain && props.domain[x] || (currentAxis === "x" && Axis.getDomain(props) || undefined,
  y: ...
}
```
So that it will 1) use the domain given to it from its parent if it exists, 2) calculate its own domain (only for its correct dimension if it isn't given a parent domain, and 3) _not_ try to calculate a domain for the opposite dimension if it's not being used within a chart because in that case it can't know anything about the opposite axis  (because it won't have access to a shared domain) and shouldn't try.